google-signin.html

```diff
@@ -33,7 +32,7 @@
 
       <paper-ripple id="ripple" class="fit"></paper-ripple>
```

Comment:
`class="fit"` (which is one of the layout classes) could be removed as well, since paper-ripple fits itself anyway